### ISSUE TITLE
loss is Nan when training with shufflenet

### ISSUE THREAD

**Issue by junedgar** [tensorboy/pytorch_Realtime_Multi-Person_Pose_Estimation#84]
Thank you for your great work!
I tried to train the model with shufflenet,  but the loss turn to Nan. Can you give me some advices?
```
Epoch: [0][0/1520]	Data time 15.252 (15.252)	Loss 0.5855 (0.5855)min_ht: -5.1922 (-5.1922)	max_paf: 4.7787 (4.7787)	min_paf: -5.5994 (-5.5994)	heatmap: 0.3290 (0.3290)	max_ht: 5.2747 (5.2747)	paf: 0.2565 (0.2565)
Epoch: [0][20/1520]	Data time 0.010 (0.772)	Loss nan (nan)min_ht: nan (nan)	max_paf: nan (nan)	min_paf: nan (nan)	heatmap: nan (nan)	max_ht: nan (nan)	paf: nan (nan)
Epoch: [0][40/1520]	Data time 0.013 (0.412)	Loss nan (nan)min_ht: nan (nan)	max_paf: nan (nan)	min_paf: nan (nan)	heatmap: nan (nan)	max_ht: nan (nan)	paf: nan (nan)
Epoch: [0][60/1520]	Data time 0.020 (0.323)	Loss nan (nan)min_ht: nan (nan)	max_paf: nan (nan)	min_paf: nan (nan)	heatmap: nan (nan)	max_ht: nan (nan)	paf: nan (nan)
Epoch: [0][80/1520]	Data time 1.571 (0.306)	Loss nan (nan)min_ht: nan (nan)	max_paf: nan (nan)	min_paf: nan (nan)	heatmap: nan (nan)	max_ht: nan (nan)	paf: nan (nan)```

**Comment by liangheng** [comment]
@junedgar Did you solve the problem? give me some advices  thankyou
